changelog.d/20251125_160002_grigorio714_remove_create_bucket_methods.md

```diff
@@ -0,0 +1,4 @@
+### Removed
+
+- Cloud storage creation no longer attempts to create bucket/container
```

Comment:
AFAICS, there was no way for the user to actually trigger this behavior, was there? In that case, we don't need a changelog entry.